airbyte-cdk/python/pyrightconfig.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What is this file used for?

Reply:
Zed, baby. Zed editor uses Pyright as python LSP, and it does not pick up the venv without this config. This file is my doing. It's not related to the rest of the PR.